docs/migration/2211_ng18/migration.md

```diff
@@ -0,0 +1,38 @@
+# (EARLY NOTES) Migrating a custom app to use Spartacus with Angular v18
+
+Before upgrading Spartacus to the new major version v2211, you need to first:
+- upgrade to the latest minor 2211.x of Spartacus
```

Comment:
We're not using the semver, so IMHO we should not use terms `major`, `minor` release.